### PR TITLE
update custom compose function to get any number of functions

### PR DESCRIPTION
**Pull request by lotgon**

### CODE REVIEW

File: 09-Function_operators.Rmd [hunk 1]
```diff
@@ -478,8 +478,8 @@
     And then we build two versions. One via an anonymous function and one via `partial()`:
     
     ```{r, eval}
-    compose_red <- function(fs) {
-      Reduce(function(f, g) function(...) f(g(...)), fs)
+    compose_red <- function(...) {
+      Reduce(function(f, g) function(...) f(g(...)), list(...))
     }
     compose_red(c(mean, length, unique))(1:10)
```
Comment (peterhurford) [inline 09-Function_operators.Rmd]: I think you now have to change it to `compose_red(mean, length, unique)`. ...The previous version also could compose any number of functions, they just had to be passed as an explicit vector.

Comment (peterhurford) [reply 09-Function_operators.Rmd]: P.S. Thanks for the contribution! 👍

**Comment by SoftFx** [comment]
fixed

**Comment by peterhurford** [comment]
Looks good to me now. I'd ping @Tazinho for final approval.

**Comment by Tazinho** [comment]
@lotgon thanks for your contribution! I will look into this the next days.
Could you please also add the phrase from https://github.com/Tazinho/Advanced-R-Solutions/blob/master/CONTRIBUTING.md

**Comment by henningsway** [comment]
Dear @lotgon,

we will merge your PR shortly, sorry for the deeeelay.

Could you please grant us the copyright for this PR? See Tazinhos Link above.

Thank you so much! :)

**Comment by lotgon** [comment]
I assign the copyright of this contribution to Malte Grosser, Henning Bumann, Robert Krzyzanowski and Peter Hurford

**Pull request closed without merging**